routes/index.js

```diff
@@ -21,27 +21,21 @@ function buildDefaultMessage(req) {
   }
 }
 
-function buildStory(req){
-  return{
-    title: 'hello',
+function buildStory(req) {
+  return {
```

Comment:
Close, but still not completely correct. Still need to incorporate what is common between all pages. Take a look again at `buildStoriesMessage` and how it uses `Object.assign`